manifests/cli_helper.pp

```diff
@@ -6,22 +6,34 @@
   include ::jenkins
   include ::jenkins::cli
 
+  $bin_path = $osfamily ? {
+    'Windows'          => '',
+    default            => '/usr/bin/',
+  }
   Class['jenkins::cli'] ->
     Class['jenkins::cli_helper'] ->
       Anchor['jenkins::end']
 
   $libdir = $::jenkins::libdir
-  $cli_jar = $::jenkins::cli::jar
+  $cli_jar = "${::jenkins::cli::jar}"
   $port = jenkins_port()
   $prefix = jenkins_prefix()
 
   $helper_groovy = "${libdir}/puppet_helper.groovy"
-  file {$helper_groovy:
-    source  => 'puppet:///modules/jenkins/puppet_helper.groovy',
-    owner   => $::jenkins::user,
-    group   => $::jenkins::group,
-    mode    => '0444',
-    require => Class['jenkins::cli'],
+  if ($::operatingsystem == 'windows')  {
+    file {$helper_groovy:
+      source  => 'puppet:///modules/jenkins/puppet_helper.groovy',
+      require => Class['jenkins::cli'],
```

Comment:
I'm not a windows user but doesn't this mean that files end up owned by the administrator?